src/views/Layout.jsx

```diff
@@ -11,19 +16,113 @@ import './Layout.css';
  * defined in `App.jsx`.
  */
 
-export function Layout() {
+export function Layout({ data, listToken, setAllDetailsOpen }) {
```

Comment:
@neshacascia  getting the prop for the current state

```suggestion
export function Layout({ data, listToken, setAllDetailsOpen, allDetailsOpen }) {
```